tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/array-test.bal

```diff
@@ -256,6 +257,11 @@ function testArrayTupleType() {
     test:assertEquals("typedesc [string[2],int,float[3][4]][][]", t.toString());
 }
 
+function testPrintByteArrayInTable() {
+    io:println(table [{ id:1, byteArray: base16 `abcd`}]);
```

Comment:
This will not test the full functionality. In order to validate full semantics we have to take the type and compare it.